system/api-server/src/app.service.ts

```diff
@@ -6,3 +6,4 @@ export class AppService {
     return 'Hello World!';
```

Comment:
This feels like boilerplate, should we remove this?

Reply:
Yeah it's from the nestjs boilerplate so we can take it out